### PR TITLE
fix:activity center view is empty

### LINKED ISSUE #4408: activity center not working in base_bc
# Bug Report

## Description

In base bc nothing appears in the activity center, including notifications, contact requests, mentions etc..

### PR DESCRIPTION
**Pull request by alexandraB99**
Closes #4408

### What does the PR do
Fixed Activity center shows nothing

### Affected areas
Activity Center

### Screenshot of functionality
ac

### CODE REVIEW

**Review by jrainville — APPROVED** [review]
Looks good. We can open an issue for the finishing touches like the community stuff, pins and reactions

**Comment by alexandraB99** [comment]
Added a few more updates and created issue with missing stuff [here](https://github.com/status-im/status-desktop/issues/4474). fyi @jrainville

**Review by osmaczko — COMMENTED** [review]
Looks good to me :) One small improvement may be applied - see code suggestion.

**Review by jrainville — APPROVED** [review]

File: ui/app/AppLayouts/Chat/stores/RootStore.qml [hunk 1]
```diff
@@ -47,7 +47,10 @@ QtObject {
     }
 
 
-    property var messageStore
+    property var messageStore: MessageStore {
+        id: messageStore
```
[the remark this answers is not in the document]
Comment (alexandraB99) [reply ui/app/AppLayouts/Chat/stores/RootStore.qml]: the MessageStore, yes, the id not. Removed id

File: ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml [hunk 2]
```diff
@@ -194,13 +195,28 @@ Popup {
...
+                            property: "previousNotificationIndex"
+                            value: notifLoader.previousNotificationIndex
+                        }
+                        Binding {
+                            target: notifLoader.item
+                            when: (notifLoader.status === Loader.Ready) && ((model.notificationType === Constants.activityCenterNotificationTypeReply)
+                                  || (model.notificationType === Constants.activityCenterNotificationTypeGroupRequest))
+                            property: "previousNotificationTimestamp"
+                            value: notifLoader.previousNotificationTimestamp
+                        }
```
[the remark this answers is not in the document]
Comment (alexandraB99) [reply ui/app/AppLayouts/Chat/popups/ActivityCenterPopup.qml]: good point, thanks! I ran it and no complains, so it seems ok.

**Review by alexandraB99 — COMMENTED** [review]
rebased and updated due to review comments

**Comment by abhi-status** [comment]
Due to the faulty Jenkins generated Windows build here I am approving this.

The Jenkins issue is now fixed on base_bc, I'll test this further there.

**Review by osmaczko — APPROVED** [review]

**Pull request merged**